docs/content/commands/npm.md

```diff
@@ -64,41 +70,44 @@ the [node-gyp Wiki](https://github.com/nodejs/node-gyp/wiki).
 
 ### Directories
 
-See [`folders`](/configuring-npm/folders) to learn about where npm puts stuff.
+See [`folders`](/configuring-npm/folders) to learn about where npm puts
+stuff.
 
 In particular, npm has two modes of operation:
 
-* global mode:
-  npm installs packages into the install prefix at
-  `prefix/lib/node_modules` and bins are installed in `prefix/bin`.
 * local mode:
   npm installs packages into the current project directory, which
-  defaults to the current working directory.  Packages are installed to
-  `./node_modules`, and bins are installed to `./node_modules/.bin`.
+  defaults to the current working directory.  packages install to
+  `./node_modules`, and bins to `./node_modules/.bin`.
+* global mode:
+  npm installs packages into the install prefix at
+  `prefix/lib/node_modules` and bins to `prefix/bin`.
```

Comment:
maybe these should be `$PREFIX/lib/node_modules` or something similar to be more clear we're referring to your global prefix and not a static directory name

Reply:
Yeah I didn't really know what to do here cause it's the `prefix` config item, literally.  `$PREFIX` is NOT the environment variable it would look for.  I suppose I could use `$npm_config_prefix`.

Reply:
that seems reasonable? or potentially drop it entirely and rephrase so it reads like:

npm installs packages in the `lib/node_modules` and bins in the `bin` directory within the configured prefix

Reply:
this is good :+1: